Lib/multiprocessing/synchronize.py

```diff
@@ -270,13 +270,13 @@ def wait(self, timeout=None):
 
     def notify(self, n=1):
         assert self._lock._semlock._is_mine(), 'lock is not owned'
-        assert not self._wait_semaphore.acquire(False)
+        assert not self._wait_semaphore.acquire(False), 'waiting to acquire'
```

Comment:
The message is rather ambiguous and unhelpful to the user.

Reply:
Could you take a look and see if my alternative phrasing is better?

Reply:
I don't think it's better. It's ok to call `notify()` even if there are no waiters. This seems to be testing an internal invariant rather than an user error.